migration/sql-files/112-cascading-delete.sql

```diff
@@ -3,17 +3,14 @@ ALTER TABLE identities
     ADD FOREIGN KEY (user_id) REFERENCES users(id) ON DELETE CASCADE,
     DROP CONSTRAINT identities_user_id_users_id_fk;
 
--- Change foreign key from "comment_revisions" to "identities" to ON DELETE CASCADE.
+-- Remove foreign key from "comment_revisions" to "identities".
 ALTER TABLE comment_revisions
-    ADD FOREIGN KEY (modifier_id) REFERENCES identities(id) ON DELETE CASCADE,
     DROP CONSTRAINT comment_revisions_identity_fk;
```

Comment:
@corinnekrych effectively this means that we need to remember by hand to which table and column the `modifier_id` column on the `comment_revisions` table points to, right?

Reply:
yes, but without removing this FK we run into a DB error when trying to delete a user.
We actually have the same relation between space and identities. ie: a owner_id without FK.